.travis.yml

```diff
@@ -12,7 +12,8 @@ install:
 - conda update -q conda
 - conda info -a
 - conda install nodejs
-- pip install cookiecutter jupyterlab
+- pip install --pre jupyterlab
```

Comment:
And this would revert before merge, eh?

Reply:
I think it is fine as-is, it will only really matter when we are mid-release anyway.